google-http-client/src/test/java/com/google/api/client/http/apache/ApacheHttpTransportTest.java

```diff
@@ -43,17 +48,16 @@ public void testApacheHttpTransport() {
     checkHttpClient(httpClient);
   }
 
-  public void testApacheHttpTransportWithParam() {
-    ApacheHttpTransport transport = new ApacheHttpTransport(new DefaultHttpClient());
-    checkHttpClient(transport.getHttpClient());
-  }
```

Comment:
Does this test need to stay deleted? Are we losing coverage of `transport.getHttpClient()`?

Reply:
No. Yes.

Thank you. A new commit just fixed that.